src/test/resources/com/axway/apim/test/files/basic/minimal-config-with-import-swagger-from-file.json

```diff
@@ -0,0 +1,9 @@
+	
+{
+	"name": "${apiName}",
+	"path": "${apiPath}",
+	"state": "${status}",
+	"version": "2.0.0",
+	"apiDefinition":"./src/test/resources/com/axway/apim/test/files/basic/petstore.json",
```

Comment:
Perhaps create only one test-file for all test-cases and make the apiDefinition line flexible in the config file. This makes it easier to maintain it and tests are easier to read as everything is at one place.
`"apiDefinition":"${testAPIDefinition",`
which is defined per test.

Reply:
ok